charts/scalardb/README.md

```diff
@@ -0,0 +1,39 @@
+# scalardb
+
+A Helm chart of Scalar DB
+Current chart version is `1.0.0`
+
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| affinity | object | `{}` | The affinity/anti-affinity feature, greatly expands the types of constraints you can express. |
+| existingSecret | string | `nil` | Name of existing secret to use for storing database username and password. |
+| fullnameOverride | string | `""` | String to fully override scalardl.fullname template |
+| nameOverride | string | `""` | String to partially override scalardl.fullname template (will maintain the release name) |
+| nodeSelector | object | `{}` | nodeSelector is form of node selection constraint. |
+| podSecurityContext | object | `{}` | PodSecurityContext holds pod-level security attributes and common container settings. |
+| replicaCount | int | `1` | Default values for number of replicas. |
+| resources | object | `{}` | Resources allowed to the pod. |
```

Comment:
I think the values other than A and B would be scalardb.xxx, but the indentation in values.yaml is wrong?

Reply:
Thank you. I'll update the indentation.